app/mutations/application/createApplicationReviewMutation.ts

```diff
@@ -0,0 +1,33 @@
+import {graphql} from 'react-relay';
+import {RelayModernEnvironment} from 'relay-runtime/lib/store/RelayModernEnvironment';
+import {
+  createApplicationReviewMutation as createApplicationReviewMutationType,
+  createApplicationReviewMutationVariables
+} from 'createApplicationReviewMutation.graphql';
+import BaseMutation from '../BaseMutation';
+
+const mutation = graphql`
+  mutation createApplicationReviewMutation(
+    $input: CreateApplicationReviewMutationChainInput!
+  ) {
+    createApplicationReviewMutationChain(input: $input) {
+      applicationReview {
+        id
+        formResultId
+        reviewStatus
+      }
+    }
+  }
+`;
+
+const createApplicationReviewMutation = async (
+  environment: RelayModernEnvironment,
+  variables: createApplicationReviewMutationVariables
+) => {
+  const m = new BaseMutation<createApplicationReviewMutationType>(
+    'create-application-review-mutation'
+  );
```

Comment:
This mutation object should be created outside of the `createApplicationReviewMutation` function so that the `clientMutationId` (defined in BaseMutation) increases. I know other mutations files currently don't do it, but they should :)